[CASB] Expand FedRAMP integrations (#21232)
cloudflare/cloudflare-docs: src/content/partials/cloudflare-one/casb/fedramp-availability.mdx

---
params:
  - integrationName
---

> :::note[Limited access]

Reviewer: ```suggestion
:::note[Availability]
```

> The {props.integrationName} CASB integration is available in limited access. To request access, contact your account team.

Reviewer: ```suggestion
The {props.integrationName} CASB integration requires a special entitlement on your account. To request access, contact your account team.
```

Reviewer: not sure if we want to call out Enterprise

Author: I think the niche use case will limit these integrations to ENT users. If someone else really needs FedRAMP, they can reach out to support.
